packages/codecs/ilp/test/benchmarks/packet.ts

```diff
@@ -1,5 +1,5 @@
 const Benchmark = require('benchmark')
-import * as PacketV1 from '../..'
```

Comment:
Ideally you want to use the `@interledger/x` import style which seems to help with the IDE stuff ... 

If you reference the files directly as here with the `src` tacked on, it means you're actually compiling the source with ts-node/register ... rather than node resolving to the lerna bootstrap symlinks ...